packages/nextjs/src/utils/initializeFronteggApp/index.ts

```diff
@@ -77,7 +77,7 @@ const initializeFronteggApp = ({
     ...options.authOptions,
     onRedirectTo,
     isLoading: false,
-    isAuthenticated: !!options.session,
+    isAuthenticated: !!options.authOptions?.isAuthenticated,
```

Comment:
this change make our admin-box think the user is authenticated before he has a jwt and causes `identity/resources/sso/v2/authenticated` to be executed and fail